files/fr/web/html/link_types/noreferrer/index.html

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Types de liens : noreferrer'
+slug: Web/HTML/Link_types/noreferrer
+browser-compat: html.elements.a.rel.noreferrer
+translation_of: 'Web/HTML/Link_types/noreferrer'
+---
+<p>Le mot-clé <strong><code>noreferrer</code></strong> de l'attribut <code><a href="/fr/docs/Web/HTML/Attributes/rel">rel</a></code> des éléments {{HTMLElement("a")}}, {{HTMLElement("area")}} et {{HTMLElement("form")}} indique au navigateur, lorsque celui-ci ouvre la ressource cible, d'omettre l'en-tête {{HTTPHeader("Referer")}} et de ne divulguer aucune information de <i lang="en">referrer</i>, comme si le mot-clé <code><a href="/fr/docs/Web/HTML/Link_types/noopener">noopener</a></code> avait également été spécifié (il n'est donc pas utile d'indiquer les deux).</p>
```

Comment:
A slight rewording which I think is necessary, to stick to the English meaning "comme si le mot-clé noopener avait également été spécifié" > "Lorsque noreferrer est indiqué, le navigateur se comporte également comme si le mot-clé noopener avait été spécifié (il n'est donc pas utile...)."

Reply:
Like in https://github.com/mdn/translated-content/pull/1799/commits/2295008695745b45ca1b02597150fd9bd2d5e809 ?